frontend/pages/my-flathub.tsx

```diff
@@ -126,12 +126,12 @@ export default function Userpage({
 
 // Need available login providers to show options on page
 export const getStaticProps: GetStaticProps = async ({ locale }) => {
-  const { data: providers } = await fetchLoginProviders()
+  const providers = await loginApi.getLoginMethodsAuthLoginGet()
```

Comment:
not a fan of the names it generates, but I guess, we could also tackle this later on. still might be easier to do now.

Reply:
Similar to the other library, this name is based on the operation name in the OpenAPI spec. It's easy to change the formula for those in the backend, but for a valid spec they should each be unique which is why FastAPI is concatenating the function name + path + HTTP method.

Another option is the method [described in the FastAPI docs](https://fastapi.tiangolo.com/advanced/generate-clients/#preprocess-the-openapi-specification-for-the-client-generator) of preprocessing the spec before feeding it to the codegen.

Reply:
I would have thought path + http method is enough to generate unique names

I guess we'll just need to rework the function naming in the backend then, probably after merging this. Stopping to include the get/post/whatever in the function name.